src/content/en/fundamentals/performance/rail.md

```diff
@@ -1,171 +1,350 @@
+      takes for the browser to paint the new frame to the screen, leaving an app about 10ms to
+      produce a frame.</td>
     </tr>
     <tr>
-      <td data-th="Delay">0 - 100ms</td>
-      <td data-th="User Reaction">Respond to a user action within this time window and users feel like the result is immediate. Any longer, and the connection between action and reaction is broken.</td>
+      <td data-th="Delay">0 to 100ms</td>
+      <td data-th="User Reaction">Respond to user actions within this time window and users
+      feel like the result is immediate. Any longer, and the connection between action and
+      reaction is broken.</td>
     </tr>
     <tr>
-      <td data-th="Delay">100 - 300ms</td>
+      <td data-th="Delay">100 to 300ms</td>
       <td data-th="User Reaction">Users experience a slight perceptible delay.</td>
     </tr>
     <tr>
-      <td data-th="Delay">300 - 1000 ms</td>
-      <td data-th="User Reaction">Within this window, things feel part of a natural and continuous progression of tasks. For most users on the web, loading pages or changing views represents a task.</td>
+      <td data-th="Delay">300 to 1000ms</td>
+      <td data-th="User Reaction">Within this window, things feel part of a natural and
+      continuous progression of tasks. For most users on the web, loading pages or changing
+      views represents a task.</td>
     </tr>
     <tr>
-      <td data-th="Delay">1000+ms</td>
-      <td data-th="User Reaction">Beyond 1 second, the user loses focus on the task they are performing.</td>
+      <td data-th="Delay">1000ms or more</td>
+      <td data-th="User Reaction">Beyond 1000 milliseconds (1 second), users lose focus on
+      the task they are performing.</td>
     </tr>
     <tr>
-      <td data-th="Delay">10,000+ms</td>
-      <td data-th="User Reaction">The user is frustrated and is likely to abandon the task; they may or may not come back later.</td>
+      <td data-th="Delay">10000ms or more</td>
+      <td data-th="User Reaction">Beyond 10000 milliseconds (10 seconds), users are frustrated
+      and are likely to abandon tasks. They may or may not come back later.</td>
     </tr>
   </tbody>
 </table>
 
-## Response: respond in under 100ms {: #response }
+Users perceive performance delays differently, depending on network conditions and hardware.
+For example, loading an experience in 1000ms is plausible on a powerful desktop machine over a
+fast Wi-Fi connection, so users have grown accustomed to a 1000ms loading experience. But for
+mobile devices over slow 3G connections, loading in 5000ms is a more realistic goal, so mobile
+users are generally more patient.
 
-You have 100ms to respond to user input before they notice a lag.
-This applies to most inputs, such as clicking buttons, toggling form
-controls, or starting animations. This does not apply to touch drags or
-scrolls.
+## Response: respond in under 50ms {: #response }
```

Comment:
@addyosmani can you review this section, I'm not sure if I captured it correctly. Transition needs to complete by 100ms? And the app needs to provide feedback within 50ms?